apps/nextjs-app/next.config.js

```diff
@@ -66,7 +66,7 @@ const tmModules = [
         // ie: '@react-google-maps/api'...
         'ky', // does not pass es-2017 checks
         'events-helsinki-components',
-        'events-helsinki-core',
+        // 'events-helsinki-core',
```

Comment:
This line can be removed all-together because the original configuration uses the core packagewithout transpiling